test/TesterInternal/GrainReferenceCastTests.cs

```diff
@@ -263,20 +263,20 @@ public void CastGrainRefUpCastFromGrandchild()
   
             // Parent
             cast = (GrainReference) grain.AsReference<IGeneratorTestDerivedGrain2>();
-            Assert.IsInstanceOfType(cast, typeof(IGeneratorTestDerivedDerivedGrain));
-            Assert.IsInstanceOfType(cast, typeof(IGeneratorTestDerivedGrain2));
-            Assert.IsInstanceOfType(cast, typeof(IGeneratorTestGrain));
+            Assert.IsType<IGeneratorTestDerivedDerivedGrain>(cast);
+            Assert.IsType<IGeneratorTestDerivedGrain2>(cast);
+            Assert.IsType<IGeneratorTestGrain>(cast);
             
             // Cross-cast outside the inheritance hierarchy should not work
-            Assert.IsNotInstanceOfType(cast, typeof(IGeneratorTestDerivedGrain1));
+            Assert.IsNotType<IGeneratorTestDerivedGrain1>(cast);
```

Comment:
not sure what the equivalent in XUnit is for IsNotInstanceOfType... there's probably none, so you'd have to use `Assert.False(cast is IGeneratorTestDerivedGrain1);`
There's other occurrences below also.